include/ftl/config.h

```diff
@@ -53,6 +53,13 @@
 	#endif
 #endif
 
+#ifdef __SSE2__
```

Comment:
https://msdn.microsoft.com/en-us/library/b0084kay.aspx

MSVC does not set this macro, even when compiling with SSE2 instructions.

Reply:
Oof. I'm basically detecting for `<emmintrin.h>` and `_mm_pause()`. I'll see if there's a better way of doing this.